Aula04/cifra.rb

```diff
@@ -0,0 +1,23 @@
+def caesar_cipher(cifra, number)
+  vetor_alfabeto = ('a'..'z').to_a
+  vetor_cifra = cifra.chars
+
+  vetor_cifra.each_with_index do |c, i|
+    vetor_alfabeto.each_with_index do |a, j|
```

Comment:
coloque nomes semânticos nas suas variaveis de loop 
```suggestion
  vetor_cifra.each_with_index do |char, index_word|
    vetor_alfabeto.each_with_index do |alpha_letter, index_alpha|
```